tests/tests/test-gas-estimation.ts

```diff
@@ -25,31 +25,15 @@ describeDevMoonbeam("Estimate Gas - Multiply", (context) => {
     ).to.equal(21994);
   });
 
+  // Since the binary search has been activated, the gas indicated in the request is not taken into
+  // account by the estimation:
+  // https://github.com/PureStake/frontier/blob/moonbeam-polkadot-v0.9.8-binary-search/client/rpc/
+  // src/eth.rs#L907
   it("should work with gas limit", async function () {
     expect(
       await multContract.methods.multiply(3).estimateGas({
-        gas: 21994,
+        gas: 0,
       })
     ).to.equal(21994);
   });
-
-  it("should ignore from balance (?)", async function () {
-    expect(
-      await multContract.methods.multiply(3).estimateGas({
-        gas: 21994,
-      })
-    ).to.equal(21994);
-  });
-
-  it("should fail with a lower gas limit", async function () {
```

Comment:
Don't remove it, rewrite it to specify it should not fail with lower gas limit (putting a comment also to why)

Reply:
Done